lib/elixir/lib/io.ex

```diff
@@ -250,7 +250,7 @@ defmodule IO do
   Reads a line from the IO device. It returns:
 
     * `data` - the characters in the line terminated
-      by a LF (or end of file)
+      by an LF (or end of file)
```

Comment:
I can see your point. It all depends on how you read it.
I read it, "the characters in the line terminated by an el-ef"

An interesting util is http://home.nerbonne.org/A-vs-An/
which scanned Wikipedia in English and created a dictionary with matches like these ones.
97 times is used, vs 17 in favor or "an LF"

but I can change it to "a", any more opinions?